interpretation/deepseismic_interpretation/azureml_pipelines/README.md

```diff
@@ -0,0 +1,175 @@
+# Integrating with AzureML
+
+## Running a Pipeline in AzureML
+Set the following environment variables:
+```
+BLOB_ACCOUNT_NAME
+BLOB_CONTAINER_NAME
+BLOB_ACCOUNT_KEY
+BLOB_SUB_ID
+AML_COMPUTE_CLUSTER_NAME
+AML_COMPUTE_CLUSTER_MIN_NODES
+AML_COMPUTE_CLUSTER_MAX_NODES
+AML_COMPUTE_CLUSTER_SKU
+```
+On Windows you can use:
+`set VARIABLE=value`
```

Comment:
![image](https://user-images.githubusercontent.com/8856246/74275683-36d09c00-4cc9-11ea-9e9a-df98d173f545.png)

Reply:
removed in 
![image](https://user-images.githubusercontent.com/8856246/74392238-521bd400-4dbb-11ea-8c8f-1cd7085aee4c.png)